main/glossary/README.md

```diff
@@ -425,20 +436,19 @@ For more information, see the [JavaScript Distributed Programming Guide](/guides
 Proposals are records with `give`, `want`, and `exit` keys. [Offers](#offer) must include a proposal, which states
 what asset you want, what asset you will give for it, and how/when the offer maker can cancel the offer
 (see [Exit Rule](#exit-rule) for details on the last). For example:
-```
+```js
 const myProposal = harden({
   give: { Asset: AmountMath.make(quatloosBrand, 4)},
   want: { Price: AmountMath.make(moolaBrand, 15) },
-  exit: { 'onDemand' }
+  exit: { onDemand: null }
 })
 ```
-`give` and `want` use [keywords](#keywords) defined by the contract. Each specifies via an [amount](#amounts), a description of what
-asset they are willing to give/want to get, and how much of it.
+`give` and `want` use [keywords](#keywords) defined by the contract. Each specifies via an [amount](#amounts) a description of what
+they are willing to give or want to get.
 
 ## Purse
-Purses hold [amounts](#amounts) of a certain [mint](#mint) issued assets. Specifically amounts that are _stationary_.
-Purses can transfer part of their held balance to a [payment](#payment), which is usually used to transfer value.
-A purse's contents are all of the same [brand](#brand).
+A purse holds [amounts](#amounts) of assets issued by a particular [mint](#mint) that are all of the same [brand](#brand) and generally _stationary_.
```

Comment:
I don't understand what "_stationary_" is supposed to mean here.

Reply:
I think it's intended as a contrast to [usually short-lived] payments. Reworded to "…often for arbitrarily long periods of time".